lib/configValidator.js

```diff
@@ -685,56 +685,57 @@ data["ignoreDuplicateSlashes"] = coerced14;
 }
 var valid0 = _errs43 === errors;
 if(valid0){
-let data13 = data.disableRequestLogging;
+let data13 = data.maxParamLength;
 const _errs45 = errors;
-if(typeof data13 !== "boolean"){
+if(!(((typeof data13 == "number") && (!(data13 % 1) && !isNaN(data13))) && (isFinite(data13)))){
+let dataType15 = typeof data13;
 let coerced15 = undefined;
 if(!(coerced15 !== undefined)){
-if(data13 === "false" || data13 === 0 || data13 === null){
-coerced15 = false;
-}
-else if(data13 === "true" || data13 === 1){
-coerced15 = true;
+if(dataType15 === "boolean" || data13 === null
+              || (dataType15 === "string" && data13 && data13 == +data13 && !(data13 % 1))){
+coerced15 = +data13;
 }
 else {
-validate10.errors = [{instancePath:instancePath+"/disableRequestLogging",schemaPath:"#/properties/disableRequestLogging/type",keyword:"type",params:{type: "boolean"},message:"must be boolean"}];
+validate10.errors = [{instancePath:instancePath+"/maxParamLength",schemaPath:"#/properties/maxParamLength/type",keyword:"type",params:{type: "integer"},message:"must be integer"}];
 return false;
 }
 }
 if(coerced15 !== undefined){
 data13 = coerced15;
 if(data !== undefined){
-data["disableRequestLogging"] = coerced15;
+data["maxParamLength"] = coerced15;
 }
 }
 }
 var valid0 = _errs45 === errors;
 if(valid0){
-let data14 = data.maxParamLength;
+let data14 = data.onProtoPoisoning;
 const _errs47 = errors;
-if(!(((typeof data14 == "number") && (!(data14 % 1) && !isNaN(data14))) && (isFinite(data14)))){
+if(typeof data14 !== "string"){
 let dataType16 = typeof data14;
 let coerced16 = undefined;
 if(!(coerced16 !== undefined)){
-if(dataType16 === "boolean" || data14 === null
-              || (dataType16 === "string" && data14 && data14 == +data14 && !(data14 % 1))){
-coerced16 = +data14;
+if(dataType16 == "number" || dataType16 == "boolean"){
+coerced16 = "" + data14;
+}
+else if(data14 === null){
+coerced16 = "";
 }
 else {
-validate10.errors = [{instancePath:instancePath+"/maxParamLength",schemaPath:"#/properties/maxParamLength/type",keyword:"type",params:{type: "integer"},message:"must be integer"}];
+validate10.errors = [{instancePath:instancePath+"/onProtoPoisoning",schemaPath:"#/properties/onProtoPoisoning/type",keyword:"type",params:{type: "string"},message:"must be string"}];
 return false;
 }
 }
 if(coerced16 !== undefined){
 data14 = coerced16;
 if(data !== undefined){
-data["maxParamLength"] = coerced16;
+data["onProtoPoisoning"] = coerced16;
 }
 }
 }
 var valid0 = _errs47 === errors;
 if(valid0){
-let data15 = data.onProtoPoisoning;
+let data15 = data.onConstructorPoisoning;
```

Comment:
I don't know why there are so many changes in this generated files after regeneration, I assume it was last generated with a different generator version